eth/downloader/downloader_test.go

```diff
@@ -1776,6 +1812,7 @@ func testFastCriticalRestarts(t *testing.T, protocol int, progress bool) {
 
 		// If it's the first failure, pivot should be locked => reenable all others to detect pivot changes
 		if i == 0 {
+			time.Sleep(150 * time.Millisecond) // Make sure no in-flight requests remain
```

Comment:
Why is this needed?

Reply:
If you look at the lines of code just below, there is a sleep directly before failing.  I found that moving the sleep to just before the explicit fail was a better place to have it, and made the test more reliable.

Reply:
@fjl Hi, just checking in to see if you saw my reply to your comment?